File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocsa/OcsaOCS0204U00MembListInfo.java
package nta.med.data.model.ihis.ocsa;

public class OcsaOCS0204U00MembListInfo {
	private String userId;
	private String userNm;
	public OcsaOCS0204U00MembListInfo(String userId, String userNm) {
		super();
		this.userId = userId;
		this.userNm = userNm;
	}
	public String getUserId() {
		return userId;
	}
	public void setUserId(String userId) {
		this.userId = userId;
	}
	public String getUserNm() {
		return userNm;
	}
	public void setUserNm(String userNm) {
		this.userNm = userNm;
	}
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocsa/OcsaOCS0204U00MembListInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocsa
{
    [Serializable]
	public class OcsaOCS0204U00MembListInfo
	{
		private String _userId;
		private String _userNm;

		public String UserId
		{
			get { return this._userId; }
			set { this._userId = value; }
		}

		public String UserNm
		{
			get { return this._userNm; }
			set { this._userNm = value; }
		}

		public OcsaOCS0204U00MembListInfo() { }

		public OcsaOCS0204U00MembListInfo(String userId, String userNm)
		{
			this._userId = userId;
			this._userNm = userNm;
		}

	}
}